index.html

```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
-      <section class="works" id="work">
-        <div class="tonic">
-          <div class="imagez">
-            <img src="images/Snapshoot-Portfolio-1.svg" alt="snap1" />
-          </div>
-          <h2>Tonic</h2>
-          <div>
-            <ul id="profs">
-              <li id="cannopy">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              A daily selection of privately personalized: no accounts or
-              sign-ups required
-            </p>
-          </div>
-          <div>
-            <ul class="button">
-              <li><button type="button">html</button></li>
-              <li><button type="button">css</button></li>
-              <li><button type="button">javascript</button></li>
-            </ul>
-          </div>
-          <div class="hidden3">
-            <button type="button">See Project</button>
-          </div>
-        </div>
-        <div class="tonic">
-          <img src="images/Snapshoot-Portfolio-2.svg" alt="snap2" />
-          <h2 id="stories-header">Multi-post Stories</h2>
-          <ul id="profs1">
-            <li id="cannopy1">CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized: no accounts or sign-ups
-            required
-          </p>
-          <ul class="button">
-            <li><button type="button">html</button></li>
-            <li><button type="button">css</button></li>
-            <li><button type="button">javascript</button></li>
-          </ul>
-          <div class="hidden1">
-            <button type="button">See Project</button>
-          </div>
-        </div>
-        <div class="tonic">
-          <div><img src="images/Snapshoot-Portfolio-3.svg" alt="snap3" /></div>
-          <div>
-            <h2 id="stories-header1">Tonic</h2>
-          </div>
-          <div>
-            <ul id="profs2">
-              <li id="cannopy2">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              A daily selection of privately personalized: no accounts or
-              sign-ups required
-            </p>
-          </div>
-          <div>
-            <ul class="button">
-              <li><button type="button">html</button></li>
-              <li><button type="button">css</button></li>
-              <li><button type="button">javascript</button></li>
-            </ul>
-          </div>
-          <div class="hidden2"><button type="button">See Project</button></div>
+      <section id="works">
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup starts from Here -->
+
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+            <div class="desktopIntro">
+              <h2 class="deskHeading"></h2>
```

Comment:
- [ ] Kindly try to make the design look as much as possible like it is in Figma. the placement of the modal elements is different, and the size of the image is different too.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190477994-cd9ccc1a-82c6-4d49-8548-3dacdc1bc45f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/190478066-8f9ee959-23eb-47df-b0c5-52ec6e908a8b.png)